lua/nvim-treesitter/install.lua

```diff
@@ -101,16 +101,30 @@ local function iter_cmd_sync(cmd_list)
   return true
 end
 
-local function run_install(cache_folder, install_folder, lang, repo, with_sync)
+local function run_install(cache_folder, install_folder, lang, repo, with_sync, generate_from_grammar)
   parsers.reset_cache()
 
   local path_sep = utils.get_path_sep()
 
   local project_name = 'tree-sitter-'..lang
+  local from_local_path = vim.fn.isdirectory(vim.fn.expand(repo.url)) == 1
+  if from_local_path then
+    repo.url = vim.fn.expand(repo.url)
```

Comment:
Could you avoid calling `expand` twice here ?

Reply:
fixed